backend/tests/Designer.Tests/GiteaIntegrationTests/GiteaIntegrationTestsBase.cs

```diff
@@ -31,16 +31,6 @@ public abstract class GiteaIntegrationTestsBase<TControllerTest> : ApiTestsBase<
         protected override string TestRepositoriesLocation =>
             Path.Combine(Path.GetTempPath(), "altinn", "tests", "repos");
 
-        /// <summary>
-        /// Used when performing chained calls to designer api
-        /// </summary>
-        protected void InvalidateAllCookies()
-        {
-            foreach (Cookie cookie in CookieContainer.GetAllCookies())
-            {
-                cookie.Expires = DateTime.Now - TimeSpan.FromHours(1);
-            }
-        }
```

Comment:
I guess this is removed in purpose, but just checking why it is not needed anymore?

Reply:
I fixed the behaviour in Gitea Delegating handler so cookies are handled there.